src/object-util/object-util.ts

```diff
@@ -20,4 +20,26 @@ export namespace ObjectUtil {
       return null;
     }
   }
+
+  export function isEmpty(value: unknown): boolean {
+    if (value == null) {
+      return true;
+    }
```

Comment:
여기 undefined도 추가해주는건 어때요?

```suggestion
if (value === null || value === undefined) {
      return true;
}
```

Reply:
네 제안해주신대로 undefined 추가하였습니다!